galley/pkg/source/kube/builtin/source_test.go

```diff
@@ -265,7 +263,10 @@ func TestPods(t *testing.T) {
 		if pod, err = client.CoreV1().Pods(namespace).Create(pod); err != nil {
 			t.Fatalf("failed creating pod: %v", err)
 		}
-		expected := toEvent(resource.Added, spec, pod, pod)
+		// multi namespace lister watcher prepends the namespace used on its creation to the original resource version
+		expectedPod := pod.DeepCopy()
```

Comment:
does it make sense to move all of these into a function?

Reply:
Yeah, it might. I just ported what we had though, did not look at refactoring. I'm happy to tackle as part of a follow-up PR